src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/util/RestHandlerUtils.kt

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
This file already existed in previous PR ?

Reply:
I don't think so, link?

Reply:
Never mind. I thought it was in previous PR but saw it here only.